context.go

```diff
@@ -149,6 +149,11 @@ func (c SpanContext) ParentID() SpanID {
 	return c.parentID
 }
 
+// Flags returns the bitmap containing such bits as 'sampled' and 'debug'.
+func (c SpanContext) Flags() byte {
+	return c.flags
```

Comment:
I think this needs to be `return c.samplingState.flags()`

Reply:
@yurishkuro Hi, I didn't find `samplingState`. Neither method or field

Reply:
make sure to sync master, it's been added recently